doc/content/devices/mac-settings/_index.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "MAC Settings"
+description: ""
+---
+
+This section provides guidelines for configuring MAC settings for end devices from the CLI.
+
+<!--more-->
+
+{{< cli-only >}}
+
+### MAC Settings and MAC State
+
+`mac-settings` on {{% tts %}} are configurable per end device. To configure persistent MAC settings, make changes to `mac-settings.desired_<parameter>`. Updates to `mac-settings.desired_<parameter>` take effect on device creation, after OTAA join or ABP FCnt reset, ResetInd, or after MAC state reset.
+
+`mac-state` can be used to test MAC settings immediately. To update settings during the current session, make changes to the `mac-state.desired_parameters.<parameter>`. Updates to the `mac-state.desired_parameters.<parameter>` are lost on reset.
+
+The expected procedure for testing and updating settings is:
+
+1. Modify `mac-state.desired_parameters.<parameter>` to see changes in the current session
+2. Test that everything works as expected
+3. Modify `mac-settings.desired_<parameter>` to make the change permanent
+
+If no settings are provided on device creation or unset, defaults are taken from [Network Server Configuration]({{< ref src="/reference/configuration/network-server" >}}).
+
+### Available MAC settings
+
+Run the following command to get a list of all available MAC settings and available parameter values:
+
+```bash
+$ ttn-lw-cli end-devices update --help
+```
+
+You can also refer to the [End Device API Reference page]({{< ref "reference/api/end_device#message:MACSettings" >}}) for documentation on the available MAC settings and MAC state parameters.
+
+### Class Specific Settings
+
+Settings that are useful based on device class are:
+
+All devices:
```

Comment:
Please split blocks with empty lines (this is a `p`, the next line is an `ul`). See also the documentation guidelines.

Same below.